### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by davidebic** [vivinano/MudaeAutoBot#87]
![image](https://user-images.githubusercontent.com/24915015/128432156-88993314-fb06-4c35-b524-dc5a67c0bdfd.png)

**Comment by vivinano** [comment]
Can I see the file that was created for the channel in question is seems like it’s unable to parse something

**Comment by davidebic** [comment]
The File is empty.
![image](https://user-images.githubusercontent.com/24915015/128508863-43a48a09-2349-462c-841a-2e0d4c13b1eb.png)

**Comment by davidebic** [comment]
Hypothesis: May it be that it cannot parse the settings message because the server is in game mode 2?

**Comment by davidebic** [comment]
Tried on a different channel in game mode one and had a similar problem
![image](https://user-images.githubusercontent.com/24915015/128511573-63911915-a2c8-4e8e-8b09-0c50a4869e2b.png)

**Comment by vivinano** [comment]
I think you are using the wrong version of discum hence unable to get the correct file. 
The parsing works on both game mode 1 and 2 no worries. So when you update discum.
Delete the empty file so it can get a valid version of the file

**Comment by davidebic** [comment]
Running the command from the discum github it tells me everything is already up to date
![image](https://user-images.githubusercontent.com/24915015/128549188-6d1f5760-c572-4981-9fad-62b0cbafb8ac.png)

**Comment by vivinano** [comment]
All right is this a new server ? Cause it tries to search for a instance of $settings if none is found then it can’t parse anything basically

**Comment by davidebic** [comment]
it's not, and also I did the $settings command before the first launch

**Comment by davidebic** [comment]
is anything else needed besides that?

**Comment by davidebic** [comment]
also, just tried restarting and now I get a different error
![image](https://user-images.githubusercontent.com/24915015/128561617-cfcbedaa-74d7-425d-aced-a14b84dd0174.png)

**Comment by vivinano** [comment]
What is the file saying now ?

**Comment by davidebic** [comment]
I deleted it to be sure but no new file was made

**Comment by davidebic** [comment]
![image](https://user-images.githubusercontent.com/24915015/128565749-8ba09833-cbaf-4424-870c-9bb47a8138b9.png)
wtf it keeps on giving different errors
I think I had seen an issue thing with illegal unicode character, lemme check

**Comment by davidebic** [comment]
![image](https://user-images.githubusercontent.com/24915015/128566461-d1884517-f7da-461a-9318-57cdba7adb39.png)
back to giving this error code

**Comment by vivinano** [comment]
Alright now there should be a file is that file empty ?

**Comment by davidebic** [comment]
yes, there is now an empty file

**Comment by vivinano** [comment]
I would just copy and paste the settings into that file somehow it isn’t writing to file

**Comment by davidebic** [comment]
Can you give me a file with the settings of another server? I'm gonna edit them manually

**Comment by davidebic** [comment]
When you have time ofc, I see you are on iPhone rn

**Comment by vivinano** [comment]
Uh on some of the closed issues there should be someone who pasted their settings file if you want to go thru that in the meantime

**Comment by davidebic** [comment]
I went through some of them but couldn't bother going through all of them. Once you are on PC just copy yours and send it to me. Let's see if it works that way.

**Comment by vivinano** [comment]
![image](https://user-images.githubusercontent.com/33008397/128620305-e40344b8-d938-44d5-9be1-64e0a30efc33.png)

**Comment by vivinano** [comment]
Haven't heard a response ? is this issue still persisting ?

**Comment by davidebic** [comment]
Manually inputting everything seems to have worked. There might be a bug with your parser. That said, thank you for the help.

TL;DR for those having the same issue, manually input the settings in the txt file in channel data

**Comment by vivinano** [comment]
The parser is unable to write emoji to a txt doc so depending on that's there could be a write issue or there a read issue and it was unable to find an instance

**Comment by vivinano** [comment]
Will be closing issue as overall this issue has been resolved.
